src/posts/2024-03-13-accelerating-cilium-adoption-with-gloo-network-for-cilium/index.md

```diff
@@ -0,0 +1,10 @@
+---
+date: '2024-03-13T18:00:00.000Z'
+title: "Solo.io's Gloo Network for Cilium is generally available"
+ogImage: ogimage.png
+ogSummary: 'Now available, Gloo Network for Cilium offers advanced observability and insights for streamlined integration across hybrid cloud environments.'
+isPopular: true
```

Comment:
```suggestion
```

removing this so that the KubeCon Paris blog still shows at the top for the event next week